chapter_08/p06_towers_of_hanoi.py

```diff
@@ -1,90 +1,120 @@
-import sys
+from abc import abstractmethod
+from collections import deque
+
+
+class Stack:
+    def __init__(self, stack_size) -> None:
+        self.stack_size = stack_size
+        self.stack = deque()
```

Comment:
I'd suggest `self._stack` to signify that this is an internal variable.

Reply:
done